### DISCUSSION TITLE
quando alguem fala o termo "hacker"

### DISCUSSION THREAD

**Discussion by camila1467** [frontendbr/forum#1656] [Discussão]
eu imagino que vocês já saibam o proposito deste post.
A minha vida toda eu sempre fui fissurada em assuntos relacionados a invasao e a hackers, por isso escolhi a profissão..
mas nunca achei realmente materiais que pudessem me ajudar realmente.. eu amo front end,acho lindo interações web mas sempre fico imaginando quantas vulnerabilidades devem existir. Eu queria saber o que de fato estudar, um guia de estudos pra poder saber se to no momento certo. AINDA NAO TENHO LINUX

**Comment by portothree** [comment]
Para vasculhar vulnerabilidades costumo visitar o [Bugzilla](https://bugzilla.mozilla.org/home), [CVE](https://www.cvedetails.com/) e [ExploitDB](https://www.exploit-db.com)
Exemplos:
https://www.exploit-db.com/exploits/19909
https://www.cvedetails.com/vulnerability-list/vendor_id-452/product_id-785/Mozilla-Bugzilla.html

**Sei que a sua pergunta foi para um guia de estudos, mas é uma boa já ir a saber quais vulnerabilidades são comuns e o que se preocupar enquanto front-end.

**Comment by barraponto** [comment]
rodar seus proprios serviços (web, banco de dados, ssh) em uma máquina virtual pode ser um começo. é pouco prático usar uma VM pra desenvolvimento, mas é um bom jeito de molhar os pés nessas águas. com o tempo vc pode adotar gerenciadores de vms (como o vagrant) ou mesmo migrar pro linux e rodar seus serviços e seu ambiente de trabalho no mesmo OS.

**Comment by jq2** [comment]
Primeiro de tudo, ... você precisa identificar o "tipo de hacker" que deseja se tornar. Atualmente, o termo "**[hacker](http://catb.org/jargon/html/H/hacker.html)**" tem muitas definições e muitas delas são definições antigas derivadas do RFC 1392, o famoso **[Jargon File](https://en.wikipedia.org/wiki/Jargon_File)** que sofreu alterações para se ajustar ao movimento open-source ("Open-source-software movement") [[Longa história ~ Hacker culture](https://en.wikipedia.org/wiki/The_Cathedral_and_the_Bazaar)].

Tudo o que posso dizer até agora é que existem hackers/mentores/programadores de olhos abertos em posts como o seu. :)

“**Diga-me com quem andas e eu te direi quem tu és**“, ... Seria interessante saber o que você procura aprender com suas palavras para que possamos ajudá-la.

**Comment by ninetails** [comment]
Não sou muito participante destes círculos porém também ressalto ter cuidado com a terminologia. Antigamente sim, hacker era aquele que estudava a fundo sistemas e vasculhava por brechas de segurança, mas mesmo nessa época, hacker também era aquele que modificava algum sistema ou hardware para benefício próprio ou de outros (sim, também se discutia bastante sobre hacker e ética). Mas na prática, sempre era aquele que estudava e sabia de tudo a ponto de encontrar tais brechas.

Uma coisa que digo é: apanhe. Não literalmente, por favor, longe disso. Estude e não pare de estudar. Comece comprando livros do assunto. Apanhe instalando e configurando Linux, pois a maioria dos sistemas web utilizam algum sistema baseado em *nix. <sup><sub><sup><sub>E depois instale o [Brave](https://brave.com/).</sub></sup></sub></sup>

Se vc for de São Paulo, visite um Hackerspace (o mais famoso por aqui é o [Garoa Hacker Clube](https://garoa.net.br/wiki/P%C3%A1gina_principal)). Dos que eu já ouvi que já frequentaram disseram que o lugar é bem integrativo.

Se você curte a cultura hacker, vá atrás de comunidades. Não lembro nenhuma de cabeça mas já vi algumas palestras na [Campus Party](https://brasil.campus-party.org/).

Quer brincar? Tem até um [playground legal](https://www.hackthebox.eu/) online para vc passar seu tempo.

Fora que vc está numa ferramenta onde é [fácil fazer `datamining` de vulnerabilidades](https://securitytrails.com/blog/github-dorks) e de infinito aprendizado.

E uma sugestão pessoal de leitura. Se vc curte esses assuntos, leia os livros do [Kevin Mitnick](https://www.amazon.com/s?k=kevin+mitnick+the+art+of) :D
